scripts/export-activity-to-rogue.php

```diff
@@ -0,0 +1,177 @@
+<?php
+/**
+ * Script to export Signups, Reportbacks, and whatever else from drupal into Rogue.
+ *
+ * to run:
+ * drush --script-path=../scripts/ php-script export-activity-to-rogue.php
+ */
+
+include_once('../lib/modules/dosomething/dosomething_rogue/dosomething_rogue.module');
+include_once('../lib/modules/dosomething/dosomething_northstar/dosomething_northstar.module');
+
+// Pick up from where we left off, if we want to
+$last_saved = variable_get('dosomething_rogue_last_signup_migrated', NULL);
+
+// Pick up getting ALL the things from where we left off!
+if ($last_saved) {
+  $signups = db_query("SELECT signups.sid, signups.uid, signups.nid, signups.run_nid, signups.source, signups.timestamp, rb.quantity, rb.why_participated, rb.rbid, rb.flagged, rb.updated
+                      FROM dosomething_signup signups
+                      LEFT JOIN dosomething_reportback rb ON signups.uid = rb.uid AND signups.nid = rb.nid AND signups.run_nid = rb.run_nid
+                      WHERE signups.sid > $last_saved AND signups.sid NOT IN (SELECT sid FROM dosomething_rogue_signups)
+                      ORDER BY signups.sid");
+}
+else {
+  // Get ALL the things!
+  $signups = db_query('SELECT signups.sid, signups.uid, signups.nid, signups.run_nid, signups.source, signups.timestamp, rb.quantity, rb.why_participated, rb.rbid, rb.flagged, rb.updated
+                      FROM dosomething_signup signups
+                      LEFT JOIN dosomething_reportback rb ON signups.uid = rb.uid AND signups.nid = rb.nid AND signups.run_nid = rb.run_nid
+                      WHERE signups.sid NOT IN (SELECT sid FROM dosomething_rogue_signups)
+                      ORDER BY signups.sid')->fetchAll();
```

Comment:
This miiiight cause trouble on production, since the `->fetchAll()` will try to load all of the signup records into an array in memory before continuing on past this line. If you just save the `db_query('...')` to `$signups` here, Drupal will use a cursor to iterate over the results in the foreach loop (so it should only load a little bit at a time).